WordPress/wporg-parent-2021
# Block styles: Add new "Brush Stroke" style for Group blocks

See https://github.com/WordPress/wporg-main-2022/issues/129, https://github.com/WordPress/wporg-showcase/issues/1 — This adds a new block style on the Group block called "Brush Stroke." When selected, it adds a mask to the group to cut out a brush stroke from the background color. A background color is needed to be visible, but any background should work.

Note: this is just for the background, the nav menu dropdown will be done separately (maybe in wporg-main-2022? not totally sure how it'll work yet).

### Screenshots

With the main child theme

| Desktop | Mobile |
|--------|-------|
| ![brush-banner](https://user-images.githubusercontent.com/541093/195454371-35ab8e83-9ace-4c1f-85f1-8a002c47cc42.png) | ![brush-banner-small](https://user-images.githubusercontent.com/541093/195454365-9264aa49-9d87-48f1-9e3f-cb2d49b28ee8.png) |

<img width="278" alt="editor" src="https://user-images.githubusercontent.com/541093/195454718-d5756a4c-3472-4c47-b2c8-0d21fd4236c8.png">

With other colors
<img width="775" alt="multiple" src="https://user-images.githubusercontent.com/541093/195455455-c3e41d1c-94ba-4664-9bb0-e1661bd3e23f.png">

### How to test the changes in this Pull Request:

1. Build the branch
2. Try using it in any post content

To test on the main child theme

1. Switch to the branch in https://github.com/WordPress/wporg-main-2022/pull/131
2. Swap out the first group in any of the download subpage patterns with the following code
3. View the pages, the blue bar should appear with the brush stroke
4. The bar should also dock to the top when scrolling, like on News (when >890px wide)

```html

<div class="wp-block-group alignfull is-style-brush-stroke has-black-color has-pomegrade-1-background-color has-text-color has-background has-link-color" style="padding-top:var(--wp--custom--brush-stroke--spacing--height);padding-right:var(--wp--preset--spacing--20);padding-bottom:0;padding-left:var(--wp--preset--spacing--20)">
<div class="wp-block-group alignwide">
<p class="has-normal-font-size" style="font-style:normal;font-weight:700;line-height:1.6">Download</p>


</div>
</div>

```
